detail/provisioning.cattle.io.cluster.vue

```diff
@@ -61,6 +61,7 @@ export default {
 
       if ( this.$store.getters['management/canList'](CAPI.MACHINE) ) {
         hash.machines = this.$store.dispatch('management/findAll', { type: CAPI.MACHINE });
+        hash.kubeNodes = this.$store.dispatch('cluster/findAll', { type: NODE });
```

Comment:
Ok, I missed this before, caught it now. This won't work as resources from the `cluster` store work in the context of the cluster used in the route (ish). Also applies to `kubeNode` in the model for `cluster.x-l8s.io.machine`.

So if you nav here from the cluster explorer's page for this cluster it'll be fine... but from any other location (say local cluster) this will fail.

I don't think there's a straight forward way to work around this, or fetch the OS for a `cluster.x-l8s.io.machine` without the kube node. Closest I got was `rke-machine.cattle.io.<provider>machinetemplate` which may contain an image, at least for DO it did.

@vincent99 Any suggestions?

Reply:
I can see where this solution breaks by navigating Explore Cluster => Local => More Resources => Cluster Provisioning => Clusters. I'm also seeing another unrelated error when working with an RKE cluster... should this area behave the same way as Cluster Management?

Reply:
Reverted these changes until we can figure out if we can get what we need from the backend